### PR TITLE
Don't log CLI actions if db not initialized

### PR DESCRIPTION
**Pull request by dstandish**
This gets rid of a potentially confusing error message.

Before:
image
After:
image

### CODE REVIEW

File: airflow/utils/cli_action_loggers.py [hunk 1]
```diff
@@ -121,8 +123,17 @@ def default_action_log(sub_command, user, task_id, dag_id, execution_date, host_
...
-    except Exception as error:
-        logging.warning("Failed to log action with %s", error)
+    except (OperationalError, ProgrammingError) as e:
+        expected = [
+            '"log" does not exist',  # postgres
+            "no such table",  # sqlite
+            "log' doesn't exist",  # mysql
+            "Invalid object name 'log'",  # mssql
+        ]
+        if getattr(e, "args", None) and not any(x in e.args[0] for x in expected):
```
Comment (uranusjr) [inline airflow/utils/cli_action_loggers.py]: I don’t think `e.args` can ever be missing (it’s guaranteed by Python)

Comment (dstandish) [reply airflow/utils/cli_action_loggers.py]: ok cool :)  i felt a bit silly writing that but i wasn't sure and wanted to err on side of safety

Comment (dstandish) [reply airflow/utils/cli_action_loggers.py]: do i even need `if e.args`?  i.e. will there always be an `e.args[0]`?

Comment (uranusjr) [reply airflow/utils/cli_action_loggers.py]: No that’s not guaranteed. `args` is just whatever init args the exception receives, so

```pycon
>>> e = Exception()
>>> e.args
()
```

Comment (dstandish) [reply airflow/utils/cli_action_loggers.py]: but i could still iterate that

Comment (dstandish) [reply airflow/utils/cli_action_loggers.py]: so this should be fine
```
        error_is_ok = any(x in e.args[0] for x in expected)
        if not error_is_ok:
            logging.warning("Failed to log action %s", e)
```
yeah?

Comment (dstandish) [reply airflow/utils/cli_action_loggers.py]: oh, no, hah (not what we're iterating)

Comment (dstandish) [reply airflow/utils/cli_action_loggers.py]: ok -- updated. i think we should be good to go here.

**Review by potiuk — APPROVED** [review]

**Pull request merged**